File: library/databindpack/src/main/java/com.loopeer.databindpack/validator/IValidator.java
package com.loopeer.databindpack.validator;

public interface IValidator {

    /**
     * check whether all Items are empty
     * @return true,the button is enable.false,the button will be disable;
     */
    boolean checkEnable();

    /**
     * check whether all items are legal. And show Toast to remind .
     * @return true,the info is legal. false,the info is illegal.
     */
    boolean isValidated();
}

File: library/validator2/src/main/java/com/loopeer/validator2/IValidator.kt
package com.loopeer.validator2

interface IValidator {

    /**
     * check whether all Items are empty
     * @return true,the button is enable.false,the button will be disable;
     */
    fun checkEnable(): Boolean

    /**
     * check whether all items are legal. And show Toast to remind .
     * @return true,the info is legal. false,the info is illegal.
     */
    fun isValidated(): Boolean
}
